src/Uno.UI/UI/Xaml/FrameworkElement.cs

```diff
@@ -438,6 +441,77 @@ private void OnRequestedThemeChanged(ElementTheme oldValue, ElementTheme newValu
 			Application.Current?.ActualElementTheme ?? ElementTheme.Default
 			: ElementTheme.Default;
 
+		public Thickness FocusVisualSecondaryThickness
```

Comment:
Those may be used quite a bit, it'd be best if they were using generated DependencyProperty

Reply:
Will do 👍